### PR TITLE
EF Migrations Drop and Data Seeding Method

### PR DESCRIPTION
**Pull request by hryhoriivoloshyn**
- All EF Migrationts Dropped 
- Added Data Seeding extension method for ModelBuilder
- Recurring Job Manager changed to add default membership to Admin

## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

- All EF Migrationts Dropped 
- Added Data Seeding extension method for ModelBuilder
- Recurring Job Manager changed to add default membership to Admin

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by bkondrat — APPROVED** [review]
good job!

File: EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj [hunk 1]
```diff
@@ -65,4 +69,8 @@
     </PackageReference>
     <PackageReference Include="NLog.Web.AspNetCore" Version="4.9.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Migrations\" />
```
Comment (tvardero) [inline EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj:74]: Навіщо міграції до сборки додавати?

**Review by tvardero — APPROVED** [review]

File: EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj (hunk 1 above)
Comment (hryhoriivoloshyn) [reply EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj]: Well, it was auto generated by ef framework or by .net project system itself. I removed these tags in the new commit and everything works fine.

Comment (hryhoriivoloshyn) [reply EPlast/EPlast.DataAccess/EPlast.DataAccess.csproj]: Here is some issues related to this auto generated tags:
https://github.com/dotnet/project-system/issues/1949
https://github.com/dotnet/efcore/issues/18381

**Review by Omeliukh-Nazar — APPROVED** [review]

**Pull request merged**